core/src/main/kotlin/transformers/descriptors/DefaultDescriptorToDocumentationTranslator.kt

```diff
@@ -70,6 +81,8 @@ class DokkaDescriptorVisitor(
             listOfNotNull(descriptorData),
             mutableSetOf() // TODO Implement following method to return proper results getXMLDRIs(descriptor, descriptorData).toMutableSet()
         )
+
+        return enum_values?.let { Enum(clazz, it) } ?: clazz
```

Comment:
Besides, what if enum has functions and properties?

Reply:
They will be copied from `clazz` object.